cypress/integration/tests_e2e/slider.spec.js

```diff
@@ -0,0 +1,52 @@
+//
+/// <reference types="cypress" />
+
+describe("Slider with assertion", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should visit slider - min and max rating", () => {
+
+    cy.get('[data-test="slider"] .ngx-slider-pointer-min')
+    .should('have.attr', 'aria-valuetext', '0')
+
+    cy.get('[data-test="slider"] .ngx-slider-pointer-max')
+    .should('have.attr', 'aria-valuetext', '5')
+  });
+
+  it("should change value of the elements", () => {
+
+    cy.get('[data-test="slider"] .ngx-slider-pointer-min')
```

Comment:
Those tests checks slider which is actually not our component.
We should test here end to end. So after change range we should check if results are correct.